lib/decision_review/schemas/review.json

```diff
@@ -193,7 +193,7 @@
                         "enum": ["RequestIssue"]
                       },
                       "id": {
-                        "type": "string"
+                        "type": ["string", "integer"]
```

Comment:
Why was this changed to include `integer`s? The response from HLR designates this as only ever being a `string` var.

Reply:
Fixed

Reply:
@camerontesterman @SMLuthi I don't have the context of the actual endpoint, but it doesn't look like it _was_ fixed

Reply:
@johnpaulashenfelter ok that is weird.. look at this commit https://github.com/department-of-veterans-affairs/vets-api/pull/3426/commits/c8166bdb8c3b0c380bcd6f4b9250747a0f5ee56f

Reply:
I did fix it and push the change... I'm not sure why it isn't reflecting that.

Reply:
That is totally weird. I checked this branch locally and it is 100% fine.